src/features/content-explorer/content-explorer/ContentExplorerIncludeSubfolders.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import Tooltip from '../../../components/tooltip';
+import Toggle from '../../../components/toggle';
+import messages from '../messages';
+import InfoBadge16 from '../../../icon/fill/InfoBadge16';
+
+const ContentExplorerIncludeSubfolders = ({ isToggleDisabled, onIncludeSubfoldersToggle, tooltipMessageForToggle }) => (
+    <div className="bdl-ContentExplorerIncludeSubfolders">
+        <div className="bdl-ContentExplorerIncludeSubfolders-toggleIconContainer">
```

Comment:
are both of these divs still necessary? can we remove `bdl-ContentExplorerIncludeSubfolders-toggleIconContainer`?
a single div inside of a div usually isn't needed